src/libs/actions/Report.js

```diff
@@ -1627,9 +1627,9 @@ function shouldShowReportActionNotification(reportID, action = null, isRemote =
         return false;
     }
 
-    // Don't show a notification if no comment exists
-    if (action && !_.some(action.message, (f) => f.type === 'COMMENT')) {
-        Log.info(`${tag} No notification because no comments exist for the current action`);
+    // Only show notifications for supported types of report actions
+    if (!ReportActionsUtils.isNotifiableReportAction(action)) {
+        Log.info(`${tag} No notification because this action type is not supported`, false, {actionName: action.actionName});
```

Comment:
This is causing some crashes:
```
Fatal Exception: com.facebook.react.common.JavascriptException: TypeError: Cannot read property 'actionName' of null, js engine: hermes, stack:
shouldShowReportActionNotification@1:3040125
shouldShowPushNotification@1:13695857
anonymous@1:13694964
anonymous@1:13688405
emit@1:162156
__callFunction@1:170848
anonymous@1:169200
__guard@1:170153
callFunctionReturnFlushedQueue@1:169158
```

If the action is not defined, we still proceed to the the block and we try to access the action.actionName which will cause crash

Reply:
created a quick PR for this here https://github.com/Expensify/App/pull/28880